spec/cli_v1_commands.yaml

```diff
@@ -567,3 +567,17 @@ commands:
       ```sh
         -h, --help   help for create
       ```
+  - id: supabase-completion
```

Comment:
Hey @lyqht - I think this file is generated by the CLI repo, so it's possible that this one will get over-written unfortunately

Should we remove this file? (or add the change to the CLI repo?

Reply:
oh i see! can you point me to how Supabase CLI generates this? if it is generated how come the supabase-completion is left out?